processor/configuration/src/main/resources/schemas/actions/catalog.json

```diff
@@ -40,5 +40,11 @@
     "name": "Webhook",
     "description": "Send the event to a webhook.",
     "isConnector": false
+  },
+  {
+    "id": "logger_sink_0.1",
+    "name": "logger",
+    "description": "Log event.",
+    "isConnector": false
```

Comment:
The user of the service has no access to the executor logs, I'd not expose it in the catalog as this is just an internal debugging detail

Reply:
Current implementation enforce us to define it in catalog else it give error.

Reply:
I agree; you cannot expose this _development_ Action in the usual way.

If it is not in the catalogue you may find how it is validated also needs refactoring.

tl;dr; adding a Logger Action is not as straight forward as you think.

Reply:
Our product not going to customer any sooner. This logger action could help developer and tester both to speed up development. 

There could be multiple ways to hide it from customer like we could create a `development_catalog.json` where we could define these type of Action. We could think of more ways.

Reply:
I agree with your reasoning; but this PR, IMO, should also include whatever "...way[s] to hide it from customer..." otherwise it will be forgotten and never addressed.